### ISSUE TITLE
Duplicate Search Button on Client Sites

### ISSUE THREAD

**Issue by nevantan** [ccswbs/hjckrrh#606]
A second search button is being generated on the search form for client sites. Example: https://aoda-client.uoguelph.ca/clerical/. This button is not visible, but is part of the tab order and effectively hides the focus indicator. Additionally, it is picked up by screenreaders (ChromeVox, NVDA in Chrome) and results in "Search Button" being announced twice.

```html
<div class="panel-pane pane-block pane-search-form">
    <h2 class="pane-title">
        Search This Site    
    </h2>
    <div class="pane-content">
        <form class="form-search content-search hideSubmitButton-processed" action="/clerical/" method="post" id="search-block-form" accept-charset="UTF-8">
            <div>
                <div>
                    <h2 class="element-invisible">Search form</h2>
                    <div class="input-group">
                        <label for="edit-search-block-form--2" class="element-invisible">Search this site</label>
                        <input title="Enter the terms you wish to search for." placeholder="Search this site" class="form-control form-text" type="text" id="edit-search-block-form--2" name="search_block_form" value="" size="15" maxlength="128">
                        <span class="input-group-btn">
                            <!-- First Search Button (Visible) -->
                            <button type="submit" class="btn btn-default">Search</button>
                        </span>
                    </div>
                    <div class="form-actions form-wrapper form-group" id="edit-actions">
                        <!-- Second Search Button (Hidden) -->
                        <button class="element-invisible btn btn-primary form-submit form-submit" type="submit" id="edit-submit" name="op" value="Search">Search</button>
                    </div>
                    <input type="hidden" name="form_build_id" value="form-bwVpzwXn_6BQQpH314tHPR53h3Zfp1ZdCe7EHz0joME">
                    <input type="hidden" name="form_token" value="9n7WSm0jlgtzKOtswAN7eoVhsZyswb8G3sC8AcnrO5k">
                    <input type="hidden" name="form_id" value="search_block_form">
                </div>
            </div>
        </form>
    </div>
</div>
```

![search_form](https://cloud.githubusercontent.com/assets/25013998/21826511/c5926bae-d755-11e6-81af-7f938ee721cb.png)

**Comment by ushenoyuog** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Duplicate Search Button on Client Sites](https://trello.com/c/J4f3ogi6/220-duplicate-search-button-on-client-sites)

**Comment by ushenoyuog** [comment]
Closing issue as it was merged !
